package.json

```diff
@@ -18,9 +18,9 @@
       "@folio/stripes-data-transfer-components"
     ],
     "okapiInterfaces": {
-      "data-import": "3.0",
-      "source-manager-job-executions": "3.0",
-      "data-import-converter-storage": "1.2"
+      "data-import": "3.1",
+      "source-manager-job-executions": "3.3",
+      "data-import-converter-storage": "1.4"
```

Comment:
Removing support for an interface version is a breaking change. Interface compatibility works like semver, so if something is compatible with 3.0 it is, by definition, also compatible with 3.1; it is not necessary to provide compatibility statements like `3.0 3.1`. 

Given the only changes since 7.0.0 was published are translations, test coverage, and lint, it seems the interfaces should have been left alone.